protocols/kad/src/behaviour.rs

```diff
@@ -2457,6 +2514,19 @@ where
             self.put_record_job = Some(job);
         }
 
+        // Poll bootstrap periodically.
+        if let Some(mut bootstrap_timer) = self.bootstrap_timer.take() {
+            if let Poll::Ready(()) = Pin::new(&mut bootstrap_timer).poll(cx) {
+                if let Some(interval) = self.bootstrap_interval {
+                    bootstrap_timer.reset(interval);
+                    if !self.any_bootstrap_successful {
+                        self.maybe_bootstrap();
+                    }
+                }
```

Comment:
I am confused. Once `any_bootstrap_successful` is set to `true`, `maybe_bootstrap` will never be called again, correct? Thus `bootstrap` is no longer called on an interval. Am I missing something?